go.mod

```diff
@@ -101,6 +101,8 @@ require (
 	oss.terrastruct.com/util-go v0.0.0-20231101220827-55b3812542c2
 )
 
+require github.com/muesli/reflow v0.3.0
```

Comment:
nit: this shouldn't be in it's own little block - but also this is the case on `main`.